lib/autoupdate/agent/setup_test.go

```diff
@@ -336,58 +411,80 @@ func TestNamespace_overrideFromConfig(t *testing.T) {
 				dataDir:          "/var/lib/teleport",
 			},
 		},
+		{
+			name: "tbot managed",
+			tbotConfig: &internal.UnversionedConfig{
+				ProxyServer: "example.com",
```

Comment:
Do we also need tests covering `AuthServer`? (which, confusingly, can contain either an auth server address _or_ proxy address).